server/integration_tests/test_data/e2e_edge_cases2/howdoesschoolsizeofurbanschoolscomparetoruralschoolsinus/chart_config.json

```diff
@@ -27,12 +27,20 @@
               {
                 "tiles": [
                   {
+                    "barTileSpec": {
```

Comment:
the data change has caused the best facet for these stat vars to only have a single data point: https://screenshot.googleplex.com/58S3k5urvoosXqC